src/Features/Core/Portable/FindUsages/DefinitionItemFactory.cs

```diff
@@ -13,47 +12,19 @@
 using Microsoft.CodeAnalysis.Features.RQName;
 using Microsoft.CodeAnalysis.FindSymbols;
 using Microsoft.CodeAnalysis.FindSymbols.Finders;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Shared.Collections;
 using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.FindUsages
 {
     using static FindUsagesHelpers;
 
-    internal interface IDefinitionsAndReferencesFactory : IWorkspaceService
-    {
-        Task<DefinitionItem?> GetThirdPartyDefinitionItemAsync(
```

Comment:
i guess we lost the override at this at some point for calling out to external parties?  if so, lovely to see this go away.

Reply:
Still calling 3rd party, just renamed this.